https://github.com/RadeonOpenCompute/ROCT-Thunk-Interface/pull/40

From fb42a02230ed5140b3ac3beadb38ca39eb9ea463 Mon Sep 17 00:00:00 2001
From: Craig Andrews <candrews@integralblue.com>
Date: Mon, 22 Jul 2019 17:17:09 -0400
Subject: [PATCH] Use CMAKE_INSTALL_PREFIX in pkgconfig

The location where files are installed is the value to use; the CPACK packaging directory isn't important as the pkgconfig file is used on the system after the package is installed.
---
 libhsakmt.pc.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libhsakmt.pc.in b/libhsakmt.pc.in
index 3093afc..1abb66d 100644
--- a/libhsakmt.pc.in
+++ b/libhsakmt.pc.in
@@ -1,4 +1,4 @@
-prefix=@CPACK_PACKAGING_INSTALL_PREFIX@
+prefix=@CMAKE_INSTALL_PREFIX@
 exec_prefix=${prefix}
 libdir=${prefix}/@CMAKE_INSTALL_LIBDIR@
 includedir=${prefix}/@CMAKE_INSTALL_INCLUDEDIR@
